BlueDBOnDisk/src/main/java/io/bluedb/disk/BlueDbOnDisk.java

```diff
@@ -63,12 +63,15 @@
 
 	@Override
 	public void backup(Path zipPath) throws BlueDbException {
+		BackupTask backupTask = new BackupTask(this, zipPath);
+		List<BlueCollectionOnDisk<?>> collectionsToBackup = getAllCollectionsFromDisk();
 		try {
-			BackupTask backupTask = new BackupTask(this, zipPath);
-			List<BlueCollectionOnDisk<?>> collectionsToBackup = getAllCollectionsFromDisk();
+			long twoHours = TimeUnit.HOURS.toMillis(2);
+			collectionsToBackup.forEach((c) -> (c).getRecoveryManager().setRetentionPeriod(twoHours));
```

Comment:
If history is cleaned up right before the default is changed will that cause a problem with the backup?